tests/e2e/utils/BrowserTabsUtil.ts

```diff
@@ -0,0 +1,74 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { Key, By, error } from 'selenium-webdriver';
+import { CLASSES } from '../inversify.types';
+import { DriverHelper } from './DriverHelper';
+import { Logger } from './Logger';
+
+@injectable()
+export class BrowserTabsUtil {
+    constructor(@inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async switchToWindow(windowHandle: string) {
```

Comment:
I, personally, don't see much value in most of these onliners. I guess the goal was to make the test more readable... but in my opinion I don't think this is that significant improvement to legitimize these methods.
I mean... 
```
browserTabsUtils.switchToWindow("sting")
```
vs 
```
driverHelper.switchTo().window("string")
```
is not a big difference (both are pretty clear to me).

If I'm the only one who's feeling this way, I'm not blocking the PR on this ;-)

Reply:
In my opinion, at least using this we have a clear steps in the logs:
```
✓ Wait debug connected (16463ms)
          ▼ BrowserTabsUtil.switchToWindow
          ▼ BrowserTabsUtil.waitContentAvailableInTheNewTab
          ▼ BrowserTabsUtil.refreshPage
      ✓ Refresh application sample window (30339ms)
          ▼ BrowserTabsUtil.switchToWindow
          ▼ Ide.waitAndSwitchToIdeFrame
          ▼ Editor.waitStoppedDebugBreakpoint title: "app.js" line: "19"
          ▼ Editor.isBreakpointPresent
```
Imagine these logs without "BrowserTabsUtil".
Besides if it possible not to use driver directly in the test,
better doesn't use it (best practices).

Reply:
Besides, your example not completely right.
There is how it will look like in real code:
```
            await browserTabsUtil.switchToWindow('string');

vs

            await (await driverHelper.getDriver()).switchTo().window('string')

```